code/style.css

```diff
@@ -1,14 +1,15 @@
 /* Global css variables used for colors */
 :root {
   --primary: #a259ff;
-  --secondary: #b0a6ff;
+  --secondary: #b0a6ff72;
 }
 
 body {
   display: flex;
   margin: 0;
   padding: 0;
   font-family: 'Montserrat', sans-serif;
+  background-image: url(https://c4.wallpaperflare.com/wallpaper/45/497/191/video-games-controllers-simple-background-playstation-wallpaper-preview.jpg);
```

Comment:
really creative background but maybe change the opacity for it to not interfere with the text.